PlaidLink.js

```diff
@@ -1,11 +1,6 @@
 import PropTypes from 'prop-types';
-import React, { useEffect } from 'react';
-import {
-  NativeEventEmitter,
-  NativeModules,
-  Platform,
-  TouchableOpacity,
-} from 'react-native';
+import React, {useEffect, useRef, useState} from 'react';
+import { Linking, NativeEventEmitter, NativeModules, Platform, TouchableOpacity } from 'react-native';
```

Comment:
nit: We might want to determine a style to avoid whitespace changes in PRs - style guide says either is fine though https://github.plaid.com/pages/plaid/docs/style/languages/typescript.html#imports

I generally run prettier in VS code

Reply:
Ran prettier :)